oas

```diff
@@ -147,39 +141,7 @@ paths:
               schema:
                 $ref: '#/components/schemas/Animal'
 
-  /feedgroups:
-    get:
-      summary: List all feedgroups
-      tags:
-        - Feedgroup Information
-      responses:
-        200:
-          description: 'successful operation'
-          content:
-            application/json:
-              schema:
-                type: array
-                items:
-                  $ref: '#/components/schemas/Feedgroup'
-  /feedgroups/{urn}:
-    get:
-      summary: Get single feedgroup
-      tags:
-        - Feedgroup Information
-      parameters:
-        - name: urn
-          in: path
-          description: urn of the Feedgroup
-          required: true
-          schema:
-            $ref: '#/components/schemas/URN'
-      responses:
-        200:
-          description: 'successful operation'
-          content:
-            application/json:
-              schema:
-                $ref: '#/components/schemas/Feedgroup'
+
 
   /subscriptions:
     post:
```

Comment:
In a fundamental service such as animal_data, I believe that support for subscriptions really should be mandatory toi avoid constant polling from all subscribers, like it is in field_data for fields/groups.

Reply:
ok, makes sense. I removed the optional keywords in descriptions and doc.